sonic-slave-buster/Dockerfile.j2

```diff
@@ -222,7 +223,7 @@ RUN apt-get update && apt-get install -y \
         libsensors4-dev \
         libwrap0-dev \
 # For lldpd
-	debhelper \
+        debhelper=13~bpo10+1 \
```

Comment:
is this a must? we had lots of issue to introduce the backported debhelper for stretch for the smartmontools build

Reply:
There are several choices:
1. build redis6: we need this as build dependency
2. directly apt install redis: no need to install debhelper, but there will be no *-dbg package

Reply:
I go with option 2 and this change is removed.